### ISSUE TITLE
Klett_BSC_NR not stored in done file list

### ISSUE THREAD

**Issue by HolgerPollyNet** [PollyNET/Pollynet_Processing_Chain#116]
I realized that the information for the Klett bsc near range is not reported in the done filelist and thus not stored to the data base.

**Comment by ZPYin** [comment]
This has been fixed.

## Bug Analysis

The figure name of Klett bsc near range plot was wrong when making done_filelist, see the code below:
https://github.com/PollyNET/Pollynet_Processing_Chain/blob/82bc73fea5b2eb5ffad9145b48ef574cfa25b890/lib/io/pollyWriteDonelist.m#L228-L229

thus, the figure was filtered out in writing the done_filelist.

**Comment by HolgerPollyNet** [comment]
Hi @ZPYin, I and @martin-rdz  realized, that no near-range Klett backscatter is calculated at all. See for example all the recent plots at the website. Only far-range bsc is plotted. Do you have any explanation why?

**Comment by ZPYin** [comment]
Regarding the near-range products, I do see both the netCDF4 files and plots in the server. Below is a sample: 

**images**
![image](https://user-images.githubusercontent.com/7610928/140950876-3c482dcf-31a5-4379-b740-0efa7ee29096.png)

**netCDF4 files**
![1636471053(1)](https://user-images.githubusercontent.com/7610928/140951820-339c86bf-414d-415d-8a45-a966bf73ff13.jpg)

In addition, the plots information has been saved in the polly DB successfully as well.
![1636471062(1)](https://user-images.githubusercontent.com/7610928/140951851-2e64a6cc-fcda-4ab8-8930-71506b92939a.jpg)

So, the only explanation why they are missing at the website is that they are just not showing!!! 
To my perspective, this is something out of Pollynet Processing Chain. Please transfer this issue to Rico (@rico-hengst ).

**Comment by HolgerPollyNet** [comment]
The problem is not, that it is not shown on the website, the problem is that the products are not calculated. I.e., the netcdf and images are there, bot no NR Klett product inside. So it is for sure a problem of the Processing Chain as it holds for all Pollys.

Arbitrary example of today:
![grafik](https://user-images.githubusercontent.com/55800803/140955257-1b5af5ab-d72f-4f25-a00b-58ea1c726391.png)

While for raman there are NR products: 
![grafik](https://user-images.githubusercontent.com/55800803/140955313-9542bb83-c3fa-4fd9-b245-b1d86b2d4d5b.png)

Any idea where to look at?

**Comment by martin-rdz** [comment]
Somehow `aerBsc_klett_NR_532` only contains masked values.

Example ``
```
:version = "3.1" ;
:history = "Last processing time at 2021-11-02 14:21:41 by pollySaveNRProfiles, 
git branch: master, git commit: 0b3e3a467de0f1f72fd979f89f7ab1a6cbe898c0" ;
data:

 aerBsc_klett_NR_532 = _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
```

Similarly for data in August 2020 (`2020_08_31_Mon_LACROS_12_00_01_1641_1759_NR_profil
es.nc`), where the overlap should have been even better.

```
:version = "3.1" ;
:history = "Last processing time at 2021-11-06 06:45:01 by pollySaveNRProfiles, 
git branch: master, git commit: 8b4e8286d58208a0b1c820b33c234a448792ca08" ;
data:

 aerBsc_klett_NR_532 = _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
```

For both instances FR and OC contains reasonable data.

**Comment by ZPYin** [comment]
Thanks for the detailed information. 

This was caused by some (stupid) coding bugs. (see below)
https://github.com/PollyNET/Pollynet_Processing_Chain/blob/8b4e8286d58208a0b1c820b33c234a448792ca08/lib/interface/picassoProcV3.m#L1232-L1235
https://github.com/PollyNET/Pollynet_Processing_Chain/blob/8b4e8286d58208a0b1c820b33c234a448792ca08/lib/interface/picassoProcV3.m#L1284-L1287

The near-field results were not transferred to the right variables...

I have fixed it in the latest fixes and updated the version to `3.2` to identify the changes. Below is the new near-field plots:

![2021_10_29_Fri_TJK_18_00_31_1800_1900_Bsc_Klett_NR](https://user-images.githubusercontent.com/7610928/141036724-7ee9ff3a-4981-476c-ac45-a08839b82a88.png)


Sorry for the inconvenience.

**Comment by HolgerPollyNet** [comment]
@ZPYin many thanks for the fast fixing and no worries about the bug - bug fixing is one of the major task while coding ;-)
